src/App.test.jsx

```diff
@@ -56,27 +44,27 @@ describe('App', () => {
       expect(queryByText('헤더')).not.toBeNull();
     });
 
-    it('shows HOME heading', () => {
+    it('shows "HOME" heading', () => {
       const { queryByText } = customRender('/');
 
       expect(queryByText('Home')).not.toBeNull();
     });
 
-    it('does not show about page description', () => {
+    it('does not show other page\'s description', () => {
```

Comment:
```suggestion
    it("does not show other page's description", () => {
```
저는 이럴경우 쌍따옴표 를 씁니다..!

Reply:
넵!! 오 이럴 때는 ESLint도 불평을 안하는군요!